### DISCUSSION TITLE
Spherical Autoencoder: mapping from SO3 to S2

### DISCUSSION THREAD

**Discussion by pablogainza** [e3nn/e3nn#394] [Q&A]
Hi ! I've been following your excellent example on the sperical MNIST database (e3nn/examples/s2cnn/mnist/). 

I'm trying to modify the architecture to turn it into an encoder/decoder. However, this requires a function that maps from an SO3 representation to an S2 representation. Do you have any suggestion of a function in the API to convert from SO3 to S2 in this context? I tried the o3.linear, but I'm not sure if that is the best.

**Comment by mariogeiger** [comment]
In Euler angle representation, if you marginalize over than gamma angle you get a function on the sphere
